addons/sourcemod/scripting/royale/config.sp

```diff
@@ -27,8 +27,8 @@ void Config_Refresh()
 	Config_ReadMapConfig(filePath);
 	
 	//Load map specific config
-	Confg_GetMapFilepath(filePath, sizeof(filePath));
-	Config_ReadMapConfig(filePath);
+	if(Config_GetMapFilepath(filePath, sizeof(filePath)))
```

Comment:
Add space between `if` and `(`

Reply:
You need to clear `filePath` or else the error message in `Config_GetMapFilepath` will never print if the global.cfg exists. This can be done by setting `filePath[0]` to `\0`.

There are other solutions to this issue but I find that the most simple.